digdag-core/src/main/java/io/digdag/core/agent/OperatorManager.java

```diff
@@ -295,7 +308,13 @@ protected TaskResult callExecutor(Path projectPath, String type, TaskRequest mer
         DefaultSecretProvider secretProvider = new DefaultSecretProvider(
                 secretContext, secretAccessPolicy, grants, operatorSecretFilter, secretStore);
 
-        TaskExecutionContext taskExecutionContext = new DefaultTaskExecutionContext(secretProvider);
+        PrivilegedVariables privilegedVariables = GrantedPrivilegedVariables.build(
+                mergedRequest.getLocalConfig().getNestedOrGetEmpty("_env"),
```

Comment:
I mean that `_env` would be used to provide a way to access secrets for any scripting operators. `py>` and `rb>` operators use scripts written by users. They need "something" to use secrets. And the "something" I found is `_env`.